README.md

```diff
@@ -235,58 +235,69 @@ For headaches, take Ibuprofen
 The gobin command installs/runs main packages.
 
 Usage:
-	gobin [-m] [-r|-p] [-n|-g] packages [run arguments...]
+	gobin [-m] [-run|-p] [-nonet|-u] packages [run arguments...]
 
-The packages argument to gobin is similar to that of the go tool (in module
-mode) with the additional constraint that the list of packages must be main
-packages.
+The packages argument to gobin is similar to that of the go get command (in
+module mode) with the additional constraint that the list of packages must be
+main packages. Each argument takes the form $main_pkg[@$version].
 
 By default, gobin is said to operate in global mode. If the -m flag is provided
 then it is said to operate in main-module mode, where the path to the main
 module's go.mod is given by go env GOMOD.
 
-The version "latest" matches the latest available tagged version. If no version
-is specified, gobin behaves differently in global and main-module modes. In
-global mode, gobin attempts to resolve the latest version available in the
-module download cache. In main-module module, gobin attempts to resolve the
-current version via the main module's go.mod. If this resolution fails in
-either mode, "latest" is assumed and gobin resolves via the network.
+The version "latest" matches the latest available tagged version for the module
+containing the main package. gobin will make a network request to resolve
+"latest" unless -nonet is provided, in which case it will attempt to resolve
+"latest" within the GOPATH module download cache. Versions that take the form
+of a revision identifier, e.g. a branch name, can only be resolved with a
+network request and hence are incompatible with -nonet.
 
-In global mode, gobin installs the main packages to the directories
-gobin/$module@$version/$main_pkg under your user cache directory. See the
-documentation for os.UserCacheDir for OS-specific details on how to configure
-its location.
-
-In main-module mode, gobin installs the main packages to the directories
-.gobincache/$module@$version/$main_pkg under the directory containing the main
-module's go.mod.
+If no version is specified for a main package, gobin behaves differently in
+global and main-module modes. In global mode, gobin attempts to resolve the
+latest tagged version available in the module download cache. In main-module
```

Comment:
perhaps mention the `-u` flag here?

    In global mode, gobin attempts to resolve the latest tagged version available in the module download cache 
    (or via the network if the `-u` flag is provided).

Reply:
Good idea. Done